### PR TITLE
Pr/fullcalendar custom header

### PR DESCRIPTION
**Pull request by niklaushug**

### CODE REVIEW

File: packages/scheduler/src/components/FullCalendar/FullCalendar.js [hunk 1]
```diff
@@ -25,6 +27,7 @@ class FullCalendar extends React.Component {
       currentStartDate: null,
       currentEndDate: null,
       wrapperId: new Date().getTime()
+
```
Comment (MethodenMann) [inline packages/scheduler/src/components/FullCalendar/FullCalendar.js]: Remove emtpy line (i know i did this :))

File: packages/scheduler/src/components/FullCalendar/FullCalendar.js [hunk 2]
```diff
@@ -125,29 +110,134 @@ class FullCalendar extends React.Component {
...
+  changeView = (view = 'timelineDay') => {
+    this.calendarElement.fullCalendar('changeView', view)
+    this.handleRangeChange()
+  }
+
+  getButtonInkProps = viewType => {
+    const currentViewType = this.calendarElement ? this.calendarElement.fullCalendar('getView').type : ''
+
+    if (currentViewType === viewType) {
```
Comment (MethodenMann) [inline packages/scheduler/src/components/FullCalendar/FullCalendar.js]: ternary operator

File: packages/scheduler/src/components/FullCalendar/FullCalendar.js [hunk 3]
```diff
@@ -125,29 +110,134 @@ class FullCalendar extends React.Component {
...
-    })
+    return {}
   }
 
-  render = () =>
-    <div className={this.state.wrapperId}><div ref={ref => { this.calendarContainer = ref }}></div></div>
+  msg = (id, values = {}) => (this.props.intl.formatMessage({id}, values))
+
+  render = () => {
+    // TODO: Remove ternary operator as soon as Child is no more a required prop
```
Comment (MethodenMann) [inline packages/scheduler/src/components/FullCalendar/FullCalendar.js]: Child is not required anymore right? Remove comment and adjust as described plase

File: packages/scheduler/src/components/FullCalendar/FullCalendar.js [hunk 4]
```diff
@@ -125,29 +110,134 @@ class FullCalendar extends React.Component {
...
+                  <Button
+                    {...(this.getButtonInkProps('timelineMonth'))}
+                    onClick={() => this.changeView('timelineMonth')}
+                  >
+                    <FormattedMessage id="client.scheduler.month"/>
+                  </Button>
+                </Menu.Item>
+              </Menu.ButtonGroup>
+            </Menu.Item>
+          </Menu.Button>
```
Comment (MethodenMann) [inline packages/scheduler/src/components/FullCalendar/FullCalendar.js]: I thint we could move this whole menu to a component called Header, what do you think?

File: packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
+import React from 'react'
```
Comment (MethodenMann) [inline packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js]: NavigationFullCalendar.spec.js? 😏 othwise well done

File: packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+            </Menu.Item>
+          </Menu.ButtonGroup>
+        </Menu.Item>
+      </Menu.Button>
+    </StyledNavigationFullCalendar>
+  )
+}
+
+NavigationFullCalendar.propTypes = {
+  changeRange: PropTypes.func,
```
Comment (MethodenMann) [inline packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js]: all this functions are required

Comment (niklaushug) [reply packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js]: yes but props.locale can be removed

Comment (MethodenMann) [reply packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js]: hm so why didnt you add .isRequired?

File: packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js (hunk 5 above)
Comment (niklaushug) [reply packages/scheduler/src/components/NavigationFullCalendar/NavigationFullCalendar.js]: done

**Pull request merged**